test/parallel/test-net-server-try-ports.js

```diff
@@ -1,52 +1,42 @@
 'use strict';
-// This tests binds to one port, then attempts to start a server on that
+// This test binds to one port, then attempts to start a server on that
 // port. It should be EADDRINUSE but be able to then bind to another port.
-require('../common');
-var assert = require('assert');
-var net = require('net');
+const common = require('../common');
+const assert = require('assert');
+const net = require('net');
 
-var server1listening = false;
-var server2listening = false;
-var server2eaddrinuse = false;
-
-var server1 = net.Server(function(socket) {
+const server1 = net.Server(function(socket) {
   socket.destroy();
```

Comment:
Instead of calling `socket.destroy()` we should just throw an error of some kind since this should never be reached. Or maybe we can at least just remove the function altogether and just have `const server1 = net.Server();`